big_tests/tests/sasl_external_SUITE.erl

```diff
@@ -317,6 +317,7 @@ self_signed_cert_is_allowed_with(EscalusTransport, C) ->
 no_cert_fails_to_authenticate(_C) ->
     UserSpec = [{username, <<"no_cert_user">>},
 		{server, <<"localhost">>},
+        {port, ct:get_config({hosts, mim, c2s_port})},
```

Comment:
Broken indentation?

Reply:
kinda.
There is a mix of tabs and whitespaces across this file.
I haven't reindent the file to reduce size of this PR.

(tldr: one tab is 4 spaces here)

Reply:
The whole file needs re-indention. 
I would not do it in this PR to simplify code review.
We can reindent in a separate PR with skip-ci.